tools/ansible/tasks/wsu/test/e2e/wsu_test.go

```diff
@@ -143,13 +144,14 @@ func testFilesCopied(t *testing.T) {
 	kubeTarPath := ansibleTempDir + "\\" + "kube.tar.gz"
 	command := fmt.Sprintf("certutil -hashfile %s SHA512", kubeTarPath)
 	stdout := new(bytes.Buffer)
-	_, err = client.Run(command, stdout, os.Stderr)
+	stderr := new(bytes.Buffer)
+	_, err = client.Run(command, stdout, stderr)
 	require.NoError(t, err, "Error generating SHA512 for %s", kubeTarPath)
+	require.Equalf(t, stderr.Len(), 0, "Error generating SHA512 for %s", kubeTarPath)
```

Comment:
Please use `require.Empty()` instead of `require.Equal` otherwise, changes lgtm!

Reply:
Here stderr isn't getting set when there's no error from command execution, so Empty check is failing. "Empty asserts that the specified object is empty. I.e. nil, "", false, 0 or either a slice or a channel with len == 0." Somehow it doesn't check for bytes.Buffer object?
That's why I am enforcing a check on the length of the stderr instead.